presto-parser/src/main/java/com/facebook/presto/sql/SqlFormatter.java

```diff
@@ -720,15 +720,14 @@ protected Void visitDelete(Delete node, Integer context)
         }
 
         @Override
-        protected Void visitCreateSchema(CreateSchema node, Integer context)
+        protected Void visitCreateSchema(CreateSchema node, Integer indent)
```

Comment:
Don't rename the parameter. It's unused and this rename is unrelated to this commit.